### PR TITLE
chore(storybook): move default values from `argTypes` to `args`

### PR DESCRIPTION
**Pull request by kevinbuhmann**
This is prep for upgrading Storybook to v8.

CDE-1847

### CODE REVIEW

File: .storybook/stories/tree/tree-node.stories.ts [hunk 1]
```diff
@@ -45,6 +41,10 @@ export default {
     onKeyDown: { control: { disable: true }, table: { disable: true } },
   },
   args: {
+    // inputs
+    clrExpandable: true,
+    clrExpanded: false,
+    clrSelected: 'not selectable',
```
Comment (valentin-mladenov) [inline .storybook/stories/tree/tree-node.stories.ts:47]: Set `clrDisabled` to `false` as well.

```suggestion
    clrDisabled: false,
    clrExpandable: true,
    clrExpanded: false,
    clrSelected: 'not selectable',
```

**Review by valentin-mladenov — COMMENTED** [review]
Check my comment.

File: .storybook/stories/tree/tree-node.stories.ts (hunk 1 above)
Comment (kevinbuhmann) [reply .storybook/stories/tree/tree-node.stories.ts:48]: Fixed.

**Review by valentin-mladenov — APPROVED** [review]
LGTM

File: .storybook/stories/tree/tree-node.stories.ts [hunk 2]
```diff
@@ -45,6 +41,11 @@ export default {
     onKeyDown: { control: { disable: true }, table: { disable: true } },
   },
   args: {
+    // inputs
+    clrDisabled: true,
```
Comment (kevinbuhmann) [inline .storybook/stories/tree/tree-node.stories.ts:45]:
```suggestion
    clrDisabled: false,
```

**Pull request merged**